streamlit_template/toolkit/queries.py

```diff
@@ -0,0 +1,166 @@
+QUERY_ENTITY_DISTRIBUTION = """
+with htan_projects as (
+    // select distinct cast(replace(NF.projectid, 'syn', '') as INTEGER) as project_id from sage.portal_raw.HTAN
+    select
+        cast(scopes.value as integer) as project_id
+    from
+        synapse_data_warehouse.synapse.node_latest,
+        lateral flatten(input => node_latest.scope_ids) scopes
+    where
+        id = 20446927
```

Comment:
It might make sense to extract this out to an environment variable so if we ever did need to change this ID we wouldn't need to create a new docker image to do it

Reply:
With the latest set of changes this was not extracted out to an environment variable. It was moved out to a global variable. What it means is that since this is a containerized application we would need to create a new image to point this at a different ID.

It isn't too big of a problem since it isn't likely to change, however, it's something to be aware of if we need to update the ID we will need to build a new image, pull, and then deploy the new image wherever this is hosted.

Reply:
Woops, my apologies for misinterpreting. For clarity: The queries that currently live in `queries.py` are actually just placeholders, and what I had in mind was that people using this template to make their own Streamlit apps would make their own queries from scratch, and use the existing examples as guides for how to construct their queries at the most.

In the future it might be a good idea to create a default `BASE_QUERY` that people can use to search for a single project, at which point we can implement a parameter like `project_id` that users can feed and get their subsetted dataframe, but I would leave that for future work.